nhsn/delphi_nhsn/patch.py

```diff
@@ -79,10 +79,11 @@ def patch(params):
     patch_list = filter_source_files(source_files)
     for file in patch_list:
         issue_date = datetime.strptime(file.name.split(".")[0], "%Y%m%d")
-        current_issue_ew = Week.fromdate(issue_date)
         logger.info("Running issue", issue_date=issue_date.strftime("%Y-%m-%d"))
         params["patch"]["issue_date"] = issue_date.strftime("%Y%m%d")
-        current_issue_dir = f"{params['patch']['patch_dir']}/issue_{current_issue_ew}/nhsn"
+        # regardless of week date type or not the directory name must be issue_date_yyyymmdd
+        # conversion in done in acquisition
+        current_issue_dir = f"{params['patch']['patch_dir']}/issue_{issue_date.strftime('%Y%m%d')}/nhsn"
```

Comment:
you use `issue_date` 3 times with `.strftime()`, but nowhere else...  is it worth making it a datetime object in the first place?  ie, would it be simpler to just do `issue_date = file.name.split(".")[0]` above?